src/Stratis.Bitcoin/Base/TimeSyncBehavior.cs

```diff
@@ -113,10 +116,10 @@ public struct TimestampOffsetSample
         /// Number of unweighted samples required before <see cref="timeOffset"/> is changed.
         /// <para>
         /// Each inbound sample counts as 1 unweighted sample.
-        /// Each outbound sample counts as <see cref="OutboundToInboundWeightRatio"/> unweighted samples.
+        /// Each outbound sample counts as <see cref="OffsetWeightSecurityConstant"/> unweighted samples.
         /// </para>
         /// </summary>
-        private const int MinUnweightedSamplesCount = 40;
+        private const int MinOutboundSampleCount = 4;
```

Comment:
Change the comment here to smth like
`Minimal amount of outbound samples that should be collected before time adjustment is allowed. `

Reply:
The para entries there aren't quite accurate now either are they

Reply:
They are not. So I'd suggest replacing all comment section for `MinOutboundSampleCount` with a single summary section with that comment I've mentioned or smth like it.

Reply:
Yep cool